kubernetes-client/src/main/java/io/fabric8/kubernetes/client/dsl/base/OperationSupport.java

```diff
@@ -601,7 +586,7 @@ protected Response retryWithExponentialBackoff(OkHttpClient client, Request requ
       } catch (IOException ie) {
         if (numRetries < requestRetryBackoffLimit) {
           retryInterval = retryIntervalCalculator.getInterval(numRetries);
-          LOG.debug(String.format("HTTP operation on url: %s should be retried after %d millis because of IOException", request.url(), retryInterval), ie);
+          LOG.debug(String.format("HTTP operation on url: %s should be retried after %d millis because of IOException", request.uri(), retryInterval), ie);
```

Comment:
Not introduced by this PR, but these placeholders `%s` should be replaced by `{}`

Reply:
I can see why this would be done, the slf4j docs should make it clearer about the convention of the last vararg being a throwable.

Reply:
OK; I also missed the String.format part :facepalm: 
But, anyway, not urgent but it should be changed to the standard approach.